conf/defaults.ini

```
@@ -288,6 +288,11 @@ application_insights_endpoint_url =
 # Controls if the UI contains any links to user feedback forms
 feedback_links_enabled = true
 
+# Defines external auth providers (such as OAuth) whos ID is used to identify to analytics services
+[analytics.external_id_auth_modules]
```

Hm, wondering about this name, it took me quite some time to understand what it was for and then only in the context of knowing the internals.

```suggestion
[analytics.trusted_auth_modules]
```

wdyt?